### DISCUSSION TITLE
Remotely Accessing Kibana Web Interface

### DISCUSSION THREAD

**Discussion by mandraid** [Security-Onion-Solutions/securityonion#1828] [Unsupported Versions] [answered]
I am having trouble remotely accessing the Kibana website.  In kibana.yml, I changed the server.host from 0 to the IP address of the machine.

**Comment by dougburks** [comment, marked as answer]
You don't need to modify kibana.yml. You should only need to run `so-allow` as described in the documentation:
https://docs.securityonion.net/en/2.3/so-allow.html#so-allow
